tech/languages/go/go-installation.md

```diff
@@ -46,19 +46,43 @@ Writing Go programs is covered in [Go programs](/tech/languages/go/go-programs.h
 
 ## Fedora Specific Notes
 
-The default installation of Go on Fedora contains two changes that Go developers should be aware of.
+The default installation of Go on Fedora contains changes to the default values of `GOPROXY`, `GOSUMDB`, and `GOTOOLCHAIN` environment variables that Go developers should be aware of. The `go` command and related tools make extensive use of environment variables for configuration. Default values are set globally in `/usr/lib/golang/go.env`, and can be overridden on per user or per project basis.
+
+For extensive help and information see:
+
+```console
+$ go help environment
+```
 
 ### GOPROXY
 
-The value of `GOPROXY` in `$GOROOT/go.env` is set to `direct`. A value of `direct` disables access to the module mirror. See [https://proxy.golang.org](https://proxy.golang.org) for more information on `GOPROXY` and the module mirror. A project specifc `go.env` can override this setting.
+The value of `GOPROXY` in `$GOROOT/go.env` is set to `direct`. A value of `direct` disables access to the module mirror. See [https://proxy.golang.org](https://proxy.golang.org) for more information on `GOPROXY` and the module mirror. A project specific `go.env` can override this setting. A user specific override can be set with:
+
+```console
+go env -w GOPROXY=https://proxy.golang.org,direct
+```
+
+### GOSUMDB
+
+The value of `GOSUMDB` in `$GOROOT/go.env` is set to `off` replacing the default value of `sum.golang.org`. The GOSUMDB environment variable identifies the name of the checksum database used to help validate downloaded modules. A project specific `go.env` can override this setting. A user specific override can be set with:
+
+```console
+go env -w GOSUMDB=sum.golang.org
```

Comment:
Good catch! Thanks.  One bit of information not in this page is any discussion about why Fedora makes these changes to the golang defaults. @alexsaezm is going to check with the original maintainer to verify the reasoning. It seems to involve a preference for a privacy centric configuration but not sure. The go-sig team may choose in the near future to change direction but in the mean time it would be good to have these settings documented here.